src/models/lazy-audio.ts

```diff
@@ -14,10 +14,16 @@ export class LazyAudio {
   constructor(private readonly audioUrl: string) {}
 
   public async play(): Promise<void> {
-    return this.loadAudio({
-      onPlayable: () => {
-        this.audio.play();
-      },
+    return new Promise<void>((resolve, reject) => {
+      this.loadAudio({
+        onPlayable: () => {
+          this.audio.addEventListener('ended', () => resolve(), { once: true });
```

Comment:
can you clarify why this is needed?

Reply:
lol, that was a whoopsies. I was playing with something that didn't work out and I forgot to delete it. 

Good catch, lol